tests/test_hyperframe.py

```diff
@@ -0,0 +1,52 @@
+import gzip
+import json
+
+import pandas as pd
+from bs4 import BeautifulSoup
+from pandas._testing import assert_frame_equal
+
+from hyperframe.hyperpattern import HyperPatternHunter
+
+EXPECTED_HYPERFRAMES_PATH = "tests/fixtures/beauhurst_expected_hyperframe_fixture.json"
+BEAUHURST_HTML_PATH = "tests/fixtures/beauhurst_scrapedpage_html.gz"
+
+
+def get_expected_hyperframes() -> list[pd.DataFrame]:
+    expected_hyperframes = []
+    with open(EXPECTED_HYPERFRAMES_PATH) as fd:
+        fixture_data = json.load(fd)
+        for frame_data, frame_columns in zip(
+            fixture_data["data"], fixture_data["columns"], strict=True
+        ):
+            # hyperframe columns are tuples, not lists (thanks json)
+            columns = [tuple(col) for col in frame_columns]
+            expected_hyperframes.append(pd.DataFrame(frame_data, columns=columns))
+    return expected_hyperframes
+
+
+def get_beauhurst_html_soup() -> BeautifulSoup:
```

Comment:
this should also be a `pytest.fixture`

Reply:
https://github.com/sam-lawrence/HyperFrame/pull/8/commits/cd5de52049305d5a0422f4e77ed05534249ca13f